pkgs/applications/editors/emacs/elisp-packages/manual-packages/gn-mode-from-sources/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  gn,
+  melpaBuild,
+}:
+
+melpaBuild {
+  pname = "gn-mode-from-sources";
```

Comment:
If `pname` is not `gn-mode`, then it is necessary to set `ename = "gn-mode";` to successfully build this package.